Integrations/integration-PagerDuty.yml

```diff
@@ -0,0 +1,475 @@
+    - name: until
+      description: The end of the date range
+    outputs:
+    - contextPath: PagerDutyUser.id
+      description: User's ID
+      type: string
+    - contextPath: PagerDutyUser.Emails
+      description: Email of user
+      type: string
+    - contextPath: PagerDutyUser.Usermame
+      description: Usermame of person
+      type: string
+    - contextPath: PagerDutyUser.DisplayName
+      description: Display name of person
+      type: string
+    - contextPath: PagerDutyUser.Role
+      description: Display role of person
+      type: string
+    description: Returns the names and details of on call users at a certain time
+      or by specific schedule
+  - name: PagerDutyGetUsersOnCallNow
+    arguments:
+    - name: query
+      description: show only the names matching the query
+    outputs:
+    - contextPath: PagerDutyUser.ID
+      description: User's ID
+      type: string
+    - contextPath: PagerDutyUser.Email
+      description: Email of user
+      type: string
+    - contextPath: PagerDutyUser.Usermame
+      description: Usermame of person
+      type: string
+    - contextPath: PagerDutyUser.DisplayName
+      description: Display name of person
+      type: string
+    - contextPath: PagerDutyUser.Role
+      description: Role of person
+      type: string
+    description: Returns the names and details of current on call personnel
+  - name: PagerDutyIncidents
+    arguments:
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - triggered
+      - acknowledged
+      - resolved
+      description: Returns only the incidents currently in the passed status(es).
+        Valid status options are triggered,acknowledged, and resolved
+    - name: since
+      description: Beginning date and time. Using ISO 8601 Representation. E.g. PagerDutyIncidents
+        since=2011-05-06T17:00Z
+    - name: sortBy
+      description: 'Used to specify both the field you wish to sort the results on,
+        as well as the direction (ascending/descending) of the results.See more https://developer.pagerduty.com/documentation/rest/incidents/list '
+    - name: until
+      description: Last date and time.  Using ISO 8601 Representation. E.g. PagerDutyIncidents
+        until=2016-05-06T13:00Z
+    outputs:
+    - contextPath: PagerDuty.Incidents
+      description: All incidents
+    description: Shows incidents in PagerDuty
+  - name: pagerDutySubmitEvent
+    arguments:
+    - name: author
+      required: true
+      description: The email address of a valid user associated with the account making
+        the request.
+    - name: title
+      required: true
+      description: The name of the incident you create
+    - name: description
+      description: A short description of the problem
+    outputs:
+    - contextPath: PagerDuty.Event
+      description: Event data
+    description: Creates a new event/incident in PagerDuty
+  isfetch: true
```

Comment:
No fetch, unclick the `fetches incidents` button

Reply:
Done